binding/SkiaSharp.Skottie/Animation.cs

```diff
@@ -22,27 +21,54 @@ void ISKNonVirtualReferenceCounted.UnreferenceNative ()
 		protected override void DisposeNative ()
 			=> SkottieApi.skottie_animation_delete (Handle);
 
-		public static bool TryParse (string data, out Animation animation)
+		public static Animation? Parse (string json) =>
+			TryParse (json, out var animation)
+				? animation
+				: null;
```

Comment:
Adding "overloads" to return the object, or null on some error that we will never know about.